app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingActivity.kt

```diff
@@ -115,10 +140,10 @@ class AddListingActivity : AppCompatActivity() {
     private fun uploadListingToServer() {
         selectedBitmap?.let { bitmap ->
             // convert bitmap to file
-            val file = File(cacheDir, "photo.png")
+            val file = File(cacheDir, "photo.jpeg")
```

Comment:
Does this integrate with S3? i.e, when we send a file, does the backend upload this and create a URL? Not sure how this is set-up? @Lonrot may know best.